client/src/js/modules/imaging/views/queuecontainer.js

```diff
@@ -592,11 +593,22 @@ define(['marionette',
             e.preventDefault()
 
             var p = this.plans.findWhere({ DIFFRACTIONPLANID: this.ui.preset.val() })
-            if (p) this.applyModel(p)
+            if (p) this.applyModel(p, false)
         },
 
-        applyModel: function(p) {
-            var models = this.qsubsamples.where({ isGridSelected: true })
+        applyPresetAll:function(e) {
+            e.preventDefault()
+
+            var p = this.plans.findWhere({ DIFFRACTIONPLANID: this.ui.preset.val() })
+            if (p) this.applyModel(p, true)
+        },
+
+        applyModel: function(p, all) {
```

Comment:
p and all are both a bit ambigous lets make it:  
  p - modelParamaeters 
  all - isLimitedToSelected
This way I think it is easier to understand the code.